### ISSUE TITLE
TypeError: Failed to fetch

### ISSUE THREAD

**Issue by enyelsequeira** [greatSumini/react-facebook-login#440]
Hello,

I am using NextJS and trying the lib to use FacebookLogin, however I am having this issue, I am not sure if this an issue with v.15 api of facebook, these are my settings 

The weird thing, is that I am able to log in and get information that is needed, but every time I click on the icon I get that error. any ideas as to why? 

![image](https://user-images.githubusercontent.com/50872387/209547756-77043d72-81b5-46d2-93f3-b733e9827533.png)

![image](https://user-images.githubusercontent.com/50872387/209547588-25614e37-c28e-4d94-aee2-a82ee2a7de46.png)

**Comment by enyelsequeira** [comment]
@greatSumini seems like  a similar issue was raised before https://github.com/keppelen/react-facebook-login/issues/349

**Comment by greatSumini** [comment]
as you think, It's the sdk's internal problem.
we cannot solve the problem directly. Instead, i have added a method to ignore.

By passing 'true' for loginOptions.ignoreSdkError, you can ignore all errors catched from sdk itself.
it's available on v3.3.1!

```tsx
<FacebookLogin
  appId="MYAPP"
  loginOptions={{
    ignoreSdkError: true,
  }}
/>
```

thanks for report 😄

**Comment by ezeev** [comment]
I'm seeing this error also on Next.js while trying it locally. The `ignoreSdkError: true` option doesn't stop it or fix it afaict.
